stack/rush-stack/src/cli/RushStackCommandLine.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { CommandLineParser } from '@microsoft/ts-command-line';
+
+import { BuildAction } from './BuildAction';
+import { CleanAction } from './CleanAction';
+
+export class RushStackCommandLine extends CommandLineParser {
+  constructor() {
+    super({
+      toolFilename: 'rush-stack',
+      toolDescription: 'The professional quality TypeScript build rig that\'s complete, integrated, and scalable'
```

Comment:
>build rig [](start = 60, length = 9)

This term has some nasty history.

Reply:
I removed it, since the command line parser is going to be redone in my next PR

---
In reply to: [179335923](https://github.com/Microsoft/web-build-tools/pull/602#discussion_r179335923) [](ancestors = 179335923)